mlir/lib/Dialect/Linalg/Transforms/BlockPackMatmul.cpp

```diff
@@ -134,6 +136,24 @@ transposePackedMatmul(RewriterBase &rewriter, linalg::LinalgOp linalgOp,
   return packTransposedMatmul;
 }
 
+static SmallVector<int64_t> getDefaultBlockFactors(linalg::LinalgOp linalgOp) {
```

Comment:
Could we slice this into a different PR to simplify reviewing?

Reply:
+1